framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/constants/DeploymentConstants.java

```diff
@@ -42,6 +42,8 @@
     public static final String DATABASE_USERNAME = "db.username";
     public static final String DATABASE_PASSWORD = "db.password";
 
+    public static final String DEFAULT_DOMAIN_SUFFIX = "default.domain.suffix";
```

Comment:
Is there any specific reason why to define domain suffix externally? Couldn't it be retrieved from org.kie.cloud.openshift.resource.Project.getDefaultRoutingSubdomain() ?

Reply:
I guess that this is required as we need to know this information when deploying Smart router together with Workbench.

Reply:
Suffix can't be get from Project, because project is deployed when is scenario deployed and we need to configured generic scenario before.